server/api/interfaces/error.go

```diff
@@ -1,6 +1,45 @@
 package interfaces
```

Comment:
why `interfaces` package. I don't see any interfaces here

Reply:
basically, `interfaces` contain WS API interfaces, but yes "interfaces" not in terms of golang

Reply:
then please rename it, it's confusing - I expected to see a bunch of interfaces, although in go we create interfaces where they're used

https://github.com/golang/go/wiki/CodeReviewComments#interfaces

Reply:
renamed